src/cpu/paging.cpp

```diff
@@ -716,9 +716,10 @@ void PAGING_LinkPage_ReadOnly(Bitu lin_page,Bitu phys_page) {
 	if (lin_page>=TLB_SIZE || phys_page>=TLB_SIZE) 
 		E_Exit("Illegal page");
 
-	if (paging.links.used>=PAGING_LINKS) {
+	if (paging.links.used >= PAGING_LINKS) {
 		LOG(LOG_PAGING,LOG_NORMAL)("Not enough paging links, resetting cache");
 		PAGING_ClearTLB();
+		assert(paging.links.used == 0);
```

Comment:
Yeah; interesting it's complaining about this! 

Even if `.used` were to become negative, the `for` loop would be skipped: 

```
	for (;paging.links.used>0;paging.links.used--)
```

However it still sets: `paging.links.used=0;` at the end, no matter what - so that's pretty cut-and-dry. 
The assert should do the trick, and absolutely improves readability as to what's going.